procedures/unit-testing-basics.ipf

```diff
@@ -2459,8 +2465,10 @@ Function RunTest(procWinList, [name, testCase, enableJU, enableTAP, enableRegExp
 		if(s.err)
 			sprintf msg, "Error %d in getTestCasesMatch: %s", s.err, s.allTestCasesList
 			UTF_PrintStatusMessage(msg)
-			sprintf msg, "Error: A test case matching the pattern \"%s\" could not be found in test suite(s) \"%s\".", s.testcase, s.procWinList
-			UTF_PrintStatusMessage(msg)
+			if(!(s.err & TC_MISSING_GENERATOR) )
```

Comment:
Here a problem might occur, because in GetTestCasesMatch the line
`err = GetRTError(1)`
can also create an error with number 64.